src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import './blocks/card';
+@import './blocks/star';
+
 body {
   margin: 0;
+  padding: 20px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), padding should be defined in a specific order: top, right, bottom, left. If `padding: 20px;` is intended to be the same for all sides, then it's correct. However, if different padding values are meant for different sides, the order needs to be specified.